index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title></title>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="./assets/css/reset.css">
+	<link rel="stylesheet" type="text/css" href="./assets/css/index.css">
+	<link href="https://fonts.googleapis.com/css?family=Nanum+Gothic|Open+Sans+Condensed:300" rel="stylesheet">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+  	<div class="img hero-section">
+
+	  		<ul>
```

Comment:
The spacing between ul on this line, div on line 12 and body tag is not consistent with tab sizing for phone list section. This is not important to address at this moment but would help to have that consistency in bigger projects that have more html :).

It is good practice to have consisten tab sizing in a single project. I personally prefer 2 space tab sizing.